### ISSUE TITLE
Node sub-resource indicator in scene tree dock

### ISSUE THREAD

**Issue by KoBeWi** [godotengine/godot#24597]
Scene tree dock contains many useful information about the nodes, like whether it's visible or has a script etc.  But now I noticed that it doesn't indicate that node has some sub-resources. You can right-click it and there's a very convenient button that shows all resources used by node. However, there's no way to tell whether the node has any resource or not until you right-click it.

I just thought that it would be nice to have some indicator. E.g. you have few labels and they use different fonts and you want to know which one has non-default font (use-case from last few minutes).

As for the indicator, there's a huge icon clutter already and sub-resource icon might be more frequent. So maybe node's name could have slightly different color when it has a sub-resource?

**Comment by Zylann** [comment]
I don't think this would help because when I look at a scene in my project, basically every node would be coloured, and nodes without resource would be a rare exception. That's an information I don't find very useful. If it was supported I would wish it was an option.

**Comment by Calinou** [comment]
Feature and improvement proposals for the Godot Engine are now being discussed and reviewed in a dedicated [**Godot Improvement Proposals (GIP)**](https://github.com/godotengine/godot-proposals) ([godotengine/godot-proposals](https://github.com/godotengine/godot-proposals)) issue tracker. The GIP tracker has a detailed issue template designed so that proposals include all the relevant information to start a productive discussion and help the community assess the validity of the proposal for the engine.

The main ([godotengine/godot](https://github.com/godotengine/godot)) tracker is now solely dedicated to bug reports and Pull Requests, enabling contributors to have a better focus on bug fixing work. Therefore, we are now closing all older feature proposals on the main issue tracker.

If you are interested in this feature proposal, please [**open a new proposal on the GIP tracker**](https://github.com/godotengine/godot-proposals/issues/new/choose) following the given issue template (after checking that it doesn't exist already). Be sure to reference this closed issue if it includes any relevant discussion (which you are also encouraged to summarize in the new proposal). Thanks in advance!
